site/building.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Building and Developing
```

Comment:
Title is building and developing but sections are Developing and Debugging.  I think these should be consistent.